Client/Frontend/Browser/Tab Tray/TabTrayController.swift

```diff
@@ -16,50 +19,124 @@ protocol TabTrayDelegate: AnyObject {
 
 class TabTrayController: LoadingViewController {
 
-  var tabTrayView: View {
-    return view as! View  // swiftlint:disable:this force_cast
-  }
-
-  override func loadView() {
-    view = View()
-  }
-
+  typealias DataSource = UICollectionViewDiffableDataSource<TabTraySection, Tab>
+  typealias Snapshot = NSDiffableDataSourceSnapshot<TabTraySection, Tab>
+  
+  // MARK: Internal
+  
   enum TabTraySection {
     case main
```

Comment:
What's the other section? I only see `main` as the only option?

Reply:
`NSDiffableDataSourceSnapshot` needs a hashable identifier and for future improvements here I would like to keep this enum.